Docs/svt-hevc_encoder_user_guide.md

```diff
@@ -342,9 +342,14 @@ Visual Studio 2017 offers Profile Guided Optimization (PGO) to improve compiler
 
 Some Linux\* Operating systems and kernels assign CPU utilization limits to applications running on servers. Therefore, to allow the application to utilize up to ~100% of the CPUs assigned to it, it is best to run the following commands before and when running the encoder:
 
->sudo  sysctl  -w  kernel.sched\_rt\_runtime\_us=1000000
+> sudo  sysctl  -w  kernel.sched\_rt\_runtime\_us=1000000
 - this command should be executed every time the server is rebooted
 
+> ./run_encode &
+  export PID=$! 
+  sudo chrt -f -a -p 99 $PIDGME
+- this command should be executed with "-rt 1" with the SVT-HEVC application to allow run-time priorities
```

Comment:
what is ./run_encoder? is $PIDGME a typo?
run encoder as background is the only way to set real-time?

Reply:
Should I use "SvtHevcEncApp -i input.mp4 -w 3840 -h 2160" instead?
$PIDGME is a typo, will change promptly.

Reply:
I think "SvtHevcEncApp -i input.**yuv** -w 3840 -h 2160" is more clearer to reader.

Reply:
Fixed